src/main/java/frc/robot/commandgroups/LaunchActuator.java

```diff
@@ -0,0 +1,37 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commandgroups;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import frc.robot.commands.CloseActuator;
+import frc.robot.commands.OpenActuator;
+
+public class LaunchActuator extends CommandGroup {
+  private static double closeTime = 0.9;
+  private static double openTime = 0.7;
+  /**
+   * Automatically launches a single frisbee
+   */
+  public LaunchActuator() {
+    addSequential(new OpenActuator(openTime));
+    addSequential(new CloseActuator(closeTime));
```

Comment:
Give a quick explanation of why you are doing sequential and what each command does.